pkg/scheduler/framework/framework.go

```diff
@@ -679,9 +679,9 @@ func (f *framework) createBindings(ctx context.Context, toCreate []*placementv1b
 							// The binding already exists, which is fine.
 							return nil
 						}
-						klog.ErrorS(err, "failed to create a new binding", "clusterResourceBinding", klog.KObj(newBinding))
+						klog.ErrorS(err, "Failed to create a new binding", "clusterResourceBinding", klog.KObj(newBinding))
 					}
-					return err
+					return controller.NewAPIServerError(false, err)
```

Comment:
we cannot wrap this error here, as the retry will need to use the original api error. Suggest to wrap the error in line 688.